<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoystickWindow</class>
 <widget class="QMainWindow" name="JoystickWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="joystickImg1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../data/data_res.qrc">:/c64_images/joystick_cropped.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="joystickImg2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../data/data_res.qrc">:/c64_images/joystick_cropped.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="c64ImgSide">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-100</y>
      <width>650</width>
      <height>284</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../data/data_res.qrc">:/c64_images/c64_side_improved.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="enabledJoy1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>joystick 1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableJoy2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>joystick 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="keyFire1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="keyFire2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="keyUp1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="keyDown1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="keyLeft1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="keyRight1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="keyLeft2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="keyUp2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="keyRight2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="keyDown2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../data/data_res.qrc"/>
 </resources>
 <connections/>
</ui>
